utils/math_functions.py

```diff
@@ -94,8 +94,8 @@ def intersection_with_circle(self, circle):
 
 	# CHECK ---->  SLOPE
 	def intersection_with_line(self, line2):
-		if not isinstance(line, line2):
-			raise ValueError("Expected Line instance, got %s" %type(line2).__name__)
```

Comment:
why this error handling is commented out?? If this is because of syntax error, `isinstance(line2, Line)` will be the correct syntax.